libraries/dawg-show/src/main/webapp/views/simplified.jsp

```diff
@@ -158,12 +133,14 @@ String fullAudioUrlOGGExtension = audioUrl + "/play1.ogg";
                 <div id="remote" class="remoteContainer" style="width=90%;">
                     <jsp:include page="<%=stdRemotePage %>" />
                 </div>
+                <% if (!isXR11) { %>
```

Comment:
Should maybe make this a more generic property. You could even store this on the Remote object like, remote.supportsMute() or something.